index.html

```diff
@@ -8,7 +8,34 @@
   </head>
 
   <body>
-    <div class="container"></div>
+    <div class="container">
+      <span class="title">To Do List</span>
+      <!-- <form action=""> -->
+      <input
+        type="text"
+        id="todoInput"
+        placeholder="할 일을 입력하세요"
+        onkeydown="EnterKeyDown()"
+      />
+      <button id="addBtn">+</button>
```

Comment:
JavaScript에서는 주로 `lowerCamelCase`를 사용하지만, CSS에서 id나 class명을 지정할 때에는 일반적으로 `kebab-case`를 사용합니다! 또한 축약형을 사용하는 것보다는 길더라도 정확한 이름을 사용하는 게 더 좋을 수도 있습니다 👀

```suggestion
      <button id="add-button">+</button>
```
<img width="870" alt="스크린샷 2024-08-28 오후 4 34 16" src="https://github.com/user-attachments/assets/85768e3c-d3e2-4311-824d-868be8569a8e">

[(참고) TOAST UI - HTML/CSS/Sass](https://ui.toast.com/fe-guide/ko_HTMLCSS#%EC%8A%A4%ED%83%80%EC%9D%BC-%EC%A7%80%EC%A0%95-%EC%8B%9C-%EC%95%84%EC%9D%B4%EB%94%94-%EB%8C%80%EC%8B%A0-%ED%81%B4%EB%9E%98%EC%8A%A4%EB%A5%BC-%EC%82%AC%EC%9A%A9%ED%95%9C%EB%8B%A4)

Reply:
자바스크립트에선 카멜케이스를 주로 사용한다고 들었는데 왜 가끔은 케밥케이스를 사용하지? 라는 의문을 가졌었는데 덕분에 궁금증이 해결됐습니다! 그리고 축약형이 무조건 좋은 줄 알았는데 아니었네요😮 피드백 감사합니다!

Reply:
컨벤션은 습관으로 들이도록 노력해야겠어요! 덕분에 복습 합니다 ㅎㅎ